### DISCUSSION TITLE
Kongapi gateway without an ingress controller

### DISCUSSION THREAD

**Discussion by yeshwanth-cmdy** [Kong/kubernetes-ingress-controller#3190] [Q&A]
How can this code be modified to disable the ingress controller?
https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/single/all-in-one-postgres.yaml

**Comment by pmalek** [comment]
Hi @yeshwanth-cmdy,

Those "all-in-one" manifests are provided to be used as is and are not really meant to be used in production environments. By that I mean that they are supported on a best effort basis.

Additionally, with the ingress controller this would basically be installing Kong Gateway (its `Deployment` and `Service`) with a bunch of other resources like KICs CRDs etc.

Having said that you can try modifying this by hand by e.g. find the ingress controller `Deployment` and `Service` and removing those.

**Comment by randmonkey** [comment]
Hi @yeshwanth-cmdy,
If you want to try deploying kong without the ingress controller, you can find the approaches to deploy it using helm in https://github.com/Kong/charts.
Here we have an example of deploying kong without ingress controller: https://github.com/Kong/charts/blob/main/charts/kong/example-values/minimal-kong-standalone.yaml
